build.gradle

```diff
@@ -50,6 +50,12 @@ if (hasProperty('buildScan')) {
     buildScan {
         termsOfServiceUrl = 'https://gradle.com/terms-of-service'
         termsOfServiceAgree = 'yes'
+        def ns = System.getenv("NAMESPACEINUSE") ?: "integration-1"
```

Comment:
Perhaps change the fallback value to an invalid namespace so we're not confused (e.g. "integration-noNS")? I'm not familiar with this env var though, so I'd appreciate your thoughts.

Reply:
Good point. I'll talk to Jeremy about this.